pyrtlnet/numpy_inference.py

```diff
@@ -201,21 +200,21 @@ def _run_layer(
         return layer_output.astype(np.int8)
 
     def preprocess_image(self, test_image: np.ndarray) -> np.ndarray:
-        """Preprocess the raw image data, as required by the quantized neural network.
+        """Preprocess the raw image data in the batch.
```

Comment:
`NumPyInference.preprocess_image` is also called by [`fpga/fpga_inference.py`](https://github.com/UCSBarchlab/pyrtlnet/blob/main/fpga/fpga_inference.py), which will also need to be updated.